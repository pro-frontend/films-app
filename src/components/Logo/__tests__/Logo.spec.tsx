import { mount, ReactWrapper } from "enzyme";
import toJson from "enzyme-to-json";
import React from "react";
import { Logo, StyledSpan } from "../";

let componentLogo: ReactWrapper, componentStyledSpan: ReactWrapper;

describe("<AppRoot />", () => {
  beforeEach(() => {
    componentLogo = mount(<Logo />);
    componentStyledSpan = mount(<StyledSpan />);
  });

  it("should mount AppRoot", () => {
    expect(toJson(componentLogo)).toMatchSnapshot();
    expect(toJson(componentStyledSpan)).toMatchSnapshot();
  });
});
