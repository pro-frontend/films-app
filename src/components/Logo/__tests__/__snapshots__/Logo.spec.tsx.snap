// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppRoot /> should mount AppRoot 1`] = `
<Logo>
  <styled.a
    href="/"
  >
    <a
      className="sc-bdfBwQ ctDQYu"
      href="/"
    >
      <b>
        netflix
      </b>
       roulette
    </a>
  </styled.a>
</Logo>
`;

exports[`<AppRoot /> should mount AppRoot 2`] = `
<styled.a>
  <a
    className="sc-bdfBwQ ctDQYu"
  />
</styled.a>
`;
